### ISSUE TITLE
When rendering to HTML any content that has 1+ styles applied is duplicated once for each style

### ISSUE THREAD

**Issue by Kethatril** [verbb/vizy#50]
**Description**
When multiple styles are added to a single piece of text (e.g. both italic and bold added to the same word) that text is duplicated once for each style. 

**For example:**
The text "anything" that is both bold and italic **_anything_** will render as `<strong>anything</strong><em>anything</em>`

It looks like the issue is caused here: https://github.com/verbb/vizy/blob/3f5e6ebc12b1d88ee7005b9eabc8cfa3f78dc4a3/src/services/Nodes.php#L125-L129 Where the text is rendered once for each mark rather then applying all the marks to the same text.

**Steps to reproduce**
You can see this in the example editor ([https://verbb.io/craft-plugins/vizy/features](https://verbb.io/craft-plugins/vizy/features)) with the word `style` which has both bold and underline applied
1. Apply more then one style to a single piece of text. 
2. View the rendered output
3. See that the text has been duplicated once for each style applied

**Additional info**
- Plugin version: 1.0.2
- Craft version: Assuming latest (3.6.17)
- Multi-site: No

**Additional context**

**Comment by engram-design** [comment]
Fixed for the next release. To get the fix early, change your `verbb/vizy` requirement in `composer.json` to:

```
"require": {
  "verbb/vizy": "dev-craft-3 as 1.0.2",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.0.3](https://github.com/verbb/vizy/releases/tag/1.0.3)
